### PR TITLE
[Keyboard] Add SIRIND Brick65S

### PR DESCRIPTION
**Pull request by eerraa**
Add SIRIND Brick65S

## Description

I would like to add SIRIND's Brick65S PCB. 
This PCB is the solder version of Brick65.
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/era/sirind/brick65s/keyboard.json [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"matrix": [4, 6], "x": 4, "y": 4.25, "w": 6.25},
+                {"matrix": [4, 11], "x": 10.5, "y": 4, "w": 1.25},
+                {"matrix": [4, 12], "x": 11.75, "y": 4, "w": 1.25},
+                {"matrix": [4, 13], "x": 13.25, "y": 4.25},
+                {"matrix": [4, 14], "x": 14.25, "y": 4.25},
+                {"matrix": [4, 15], "x": 15.25, "y": 4.25}
+            ]
+        },
+        "LAYOUT_brick65_ansi": {
```
Comment (zvecr) [inline keyboards/era/sirind/brick65s/keyboard.json:111]: Layout macros should not contain the keyboard name.
```suggestion
        "LAYOUT_ansi": {
```

File: keyboards/era/sirind/brick65s/keyboard.json [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"matrix": [4, 6], "x": 4, "y": 4.25, "w": 6.25},
+                {"matrix": [4, 11], "x": 10.5, "y": 4, "w": 1.25},
+                {"matrix": [4, 12], "x": 11.75, "y": 4, "w": 1.25},
+                {"matrix": [4, 13], "x": 13.25, "y": 4.25},
+                {"matrix": [4, 14], "x": 14.25, "y": 4.25},
+                {"matrix": [4, 15], "x": 15.25, "y": 4.25}
+            ]
+        },
+        "LAYOUT_brick65_ansi_split_bs": {
```
Comment (zvecr) [inline keyboards/era/sirind/brick65s/keyboard.json:180]:
```suggestion
        "LAYOUT_ansi_split_bs": {
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/era/sirind/brick65s/keyboard.json (hunk 2 above)
Comment (eerraa) [reply keyboards/era/sirind/brick65s/keyboard.json]: @zvecr 
Oh, I missed that.
Thank you for your help.

**Comment by eerraa** [comment]
@dunk2k @drashna @waffle87 
Hello sir,
Can I ask why the QMK merge of this PR is delayed?

File: keyboards/era/sirind/brick65s/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_all(
+        KC_ESC,     KC_1,     KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS,  KC_EQL,   KC_BSLS,   KC_BSPC,   KC_INS,
+        KC_TAB,     KC_Q,     KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC,  KC_RBRC,             KC_BSLS,   KC_DEL,
+        KC_CAPS,    KC_A,     KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,            KC_ENT,
+        KC_LSFT,              KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT,             KC_UP, 
+        KC_LCTL,    KC_LWIN,  KC_LALT,                            KC_SPC,                             KC_RALT, KC_RCTL,            KC_LEFT,   KC_DOWN,   KC_RGHT
+    ),
+    [1] = LAYOUT_all(
```
Comment (waffle87) [inline keyboards/era/sirind/brick65s/keymaps/default/keymap.c:14]: This layer does not appear to be accessible.

Comment (eerraa) [reply keyboards/era/sirind/brick65s/keymaps/default/keymap.c:14]: I don't understand the meaning. Is there a reason why it is not accessible?

Comment (waffle87) [reply keyboards/era/sirind/brick65s/keymaps/default/keymap.c:14]: Because there is no keycode on layer 0 that goes to this layer.

Comment (eerraa) [reply keyboards/era/sirind/brick65s/keymaps/default/keymap.c:14]: Thanks for letting me know. I fixed it.

**Review by waffle87 — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**